packages/@roots/bud-purgecss/src/index.ts

```diff
@@ -4,39 +4,49 @@
 /**
  * Adds purgecss support to Bud
  *
- * @remarks
- * Requires {@link @roots/bud-postcss# | @roots/bud-postcss} to be installed
+ * @example
+ * ```ts
+ * app.purge({
+ *  content: [app.path('project', 'resources/views/**')],
+ *  allow: require('purgecss-with-wordpress').whitelist,
+ *  allowPatterns: require('purgecss-with-wordpress').whitelistPatterns,
+ * })
+ * ```
  *
  * @see https://roots.io/bud
  * @see https://github.com/roots/bud
  *
- * @remarks
- * - 💁 Composable - Build exceptional applications with a modular, configurable build system
- *
- * - 💪 Modern - Modern framework written in TypeScript with an expressive API
- *
- * - 🌱 Easy - Low bundle size and fast build times
- *
  * @packageDocumentation
  */
 
-import {Extension} from '@roots/bud-framework'
+import './bud.env'
 
-import {purge} from './bud.purge'
+import {purgecss} from './bud.purge'
+import {Purge} from './purge.interface'
 
-declare module '@roots/bud-framework' {
-  interface Framework {
-    purge: purge
-  }
+/**
+ * Module name
+ *
+ * @public
+ */
+export const name: Purge['name'] = '@roots/bud-purgecss'
 
-  /**
-   * {@inheritDoc @roots/bud-framework#Modules}
-   * @public @override
-   */
-  interface Modules {
-    '@roots/bud-purgecss': Extension.Module
-  }
-}
+/**
+ * Module api
+ *
+ * @public
+ */
+export const api: Purge['api'] = {purgecss}
 
-export const name = '@roots/bud-purgecss'
-export const api = {purge}
+/**
+ * Module registration
+ *
+ * @todo facade bindings to `@roots/bud-extension`
```

Comment:
The only reason for this `register` callback is that I need to set/bind the facade for `bud.config.js`. 

this should eventually be moved to `@roots/bud-extensions/src/Controller/controller.service.ts`

Reply:
addressing this in #906. let's get that merged first and then I can remove this.